doc/developers-guide/accumulators.md

```diff
@@ -12,11 +12,11 @@ At the beginning of the main processing chain an accumulator is created containi
 * `origin_stanza` - Original stanza that triggered the processing (in a binary).
 * `lserver` - Nameprepped domain of the processing context.
 * `stanza` - A map with information about the stanza being routed. May be missing in some processing chains (when they are not triggered by a stanza)!
-  * `element` - `exml:element()` with the current stanza being routed.
-  * `from_jid`, `to_jid` - `jid:jid()` with the sender and the recipient.
-  * `name` - A name of the top-level element in `element`.
-  * `type` - A value of `type` attribute of the top-level element. If the attribute is missing, this field contains `undefined`.
-  * `ref` - A reference of routed stanza.
+    * `element` - `exml:element()` with the current stanza being routed.
```

Comment:
Why here is added extra spaces? Looks like 2 spaces should work.

* No space
  * 2 spaces
    *  4 spaces

Reply:
That's a good question! However, I learned there are a few differences in Markdown, so GH will render stuff differently sometimes. We use MkDocs, which uses python-markdown to render markdown. This in turn insists on 4 spaces for indentation: https://python-markdown.github.io/#differences
So this was not a sublist, and rendered at the top level: https://esl.github.io/MongooseDocs/latest/developers-guide/accumulators/

Reply:
Oh, I see, thanks for info.